### DISCUSSION TITLE
NEEY 15A Smart Active Balancer: confirmed as working :)

### DISCUSSION THREAD

**Discussion by msevestre31** [syssi/esphome-jk-bms#487] [General]
Hello

I can confirm the powerful (yet, expensive) NEEY 15A active balancer works perfectly with the project (configured the same was as the 4A flavor).

**Comment by syssi** [comment]
Thanks for your feedback! Could you provide the software and hardware version of your device?

**Reply by msevestre31** [reply]
Unfortunately, sorry no - this is a test I did this weekend with a brand-new unit, which I sent back this morning to its reseller (15A was finally overkill for my needs)

**Reply by msevestre31** [reply]
EK-24S15EB is the only reference I can provide, without any further HW/SW version :(

**Comment by chipsi** [comment]
I can confirm NEEY 10A active balancer works. I added GW- and EK- BLE signature. EK-24S10EB, HW version : HW-1.1.5, Software version : SW-1.1.1. On PCB text : Enerkey EK-24S8EB V1.5 @syssi
